components/logins/src/db.rs

```diff
@@ -121,7 +139,13 @@ impl Deref for LoginDb {
 // login specific stuff.
 
 impl LoginDb {
-    fn mark_as_synchronized(&self, guids: &[&str], ts: ServerTimestamp) -> Result<()> {
+    fn mark_as_synchronized(
+        &self,
+        guids: &[&str],
+        ts: ServerTimestamp,
+        scope: &SqlInterruptScope,
+    ) -> Result<()> {
+        let tx = self.unchecked_transaction()?;
```

Comment:
Oh! This wasn't running a transaction before?

Reply:
Nope, Certainly should have been though!